casual/casual-api/src/main/java/se/laz/casual/jca/DomainId.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2022, The casual project. All rights reserved.
+ *
+ * This software is licensed under the MIT license, https://opensource.org/licenses/MIT
+ */
+
+package se.laz.casual.jca;
+
+import java.io.Serializable;
+import java.util.Objects;
+import java.util.UUID;
+
+public class DomainId implements Serializable
```

Comment:
Do we really need a whole object to represent a just a UUID?

Reply:
Yes :)
The type is clearly what it is, a DomainId - so it does not matter what actual type is used internally, even if changed it still remains a DomainId.